04-mixing.md

```diff
@@ -2,6 +2,9 @@
 How To Clear - Mixing content and creating updates
 ==================================================
 
+For official documentation on these topics please see
+[https://clearlinux.org/documentation](clearlinux.org/documentation).
```

Comment:
Should be a link to https://clearlinux.org/documentation/clear-linux/guides/maintenance/mixer directly.

Reply:
done